webgui/scripts/codecompass/view/fileManager.js

```diff
@@ -237,6 +239,46 @@ function (on, query, style, topic, declare, Memory, Observable,
       path.unshift('project');
 
       this.set('path', path);
+      this.set('filePath', fileInfo.path);
+    },
+
+    _setFilePathAttr : function (path) {
+      this._filePath = path;
+      this._setFilePath();
+    },
+
+    /*
+     * Calculate the actual width of an element
+     */
+    getWidthOfAnElement : function(node){
```

Comment:
This should be a "private" function, i.e. user underscore prefix. It would even be better to place this to a more global place in the source code (maybe in the utils?). See also the `Text` module which has a function to compute the height of an element: https://github.com/Ericsson/CodeCompass/blob/master/webgui/scripts/codecompass/view/component/Text.js#L136. Can't these two functions have the same structure? That would be one more reason to place both of these to a util file.